### PR TITLE
Fix CI pipeline

### LINKED ISSUE #5: Fix checks
Make sure that all checks pass.

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
**Reference to a related issue in the repository**
#5 

**Add a description**
- [x] Fix CPP and Markdown linter remarks
- [x] Add new trace file for smoke test

**Take this checklist as orientation for yourself, if this PR is ready for Maintainer Review**
- [x] My suggestion follows the [governance rules](https://openmsl.github.io/doc/OpenMSL/organization/governance_rules.html).
- [ ] All commits of this PR are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
- [x] My changes generate no errors when passing CI tests. 
- [x] I updated all documentation (readmes incl. figures) according to my changes.
- [x] I have successfully implemented and tested my fix/feature locally.
- [ ] Appropriate reviewer(s) are assigned.

### CODE REVIEW

**Comment by ClemensLinnhoff** [comment]
@kristofhfr I fixed the linter problems. Now only the smoke test fails because of #3. Can you add the new trace file on this branch?

**Review by kristofhfr — APPROVED** [review]

File: test/integration/001_smoke_test_tracefile/20241028T112405Z_sv_360_2112_1.osi
Comment (ClemensLinnhoff) [inline test/integration/001_smoke_test_tracefile/20241028T112405Z_sv_360_2112_1.osi:1]: All time stamps in this file seem to be 0.

File: README.md
Comment (ClemensLinnhoff) [inline README.md:1]: Please add Readme with the structure given in the template repository.

**Comment by ClemensLinnhoff** [comment]
The [readme in the integration test folder](https://github.com/openMSL/sl-1-5-reflection-based-fmcw-lidar-detection-model/blob/5-fix-checks/test/integration/README.md) is not up-to-date.

**Comment by ClemensLinnhoff** [comment]
The [readme in the smoke test folder](https://github.com/openMSL/sl-1-5-reflection-based-fmcw-lidar-detection-model/tree/5-fix-checks/test/integration/001_smoke_test_tracefile) is not up-to-date.

File: test/integration/001_smoke_test_tracefile/20241028T112405Z_sv_360_2112_1.osi
Comment (kristofhfr) [reply test/integration/001_smoke_test_tracefile/20241028T112405Z_sv_360_2112_1.osi:1]: To keep the size of the file to a minimum it only contains one frame, is that a problem?

Comment (ClemensLinnhoff) [reply test/integration/001_smoke_test_tracefile/20241028T112405Z_sv_360_2112_1.osi:1]: Ah I understand. I think I would still do at least two time frames, so you know that the model can handle a time sequence of frames.

**Comment by ClemensLinnhoff** [comment]
Please fix your DCO.

**Comment by ClemensLinnhoff** [comment]
An please fill out the checklist in the PR description.

**Pull request merged**